EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/audience_query.json

```diff
@@ -0,0 +1,45 @@
+[
```

Comment:
Isn't this an allocation? Not specifically an audience query?

Reply:
Exactly. Will be removed